src/switch_telnyx.cpp

```diff
@@ -95,6 +98,16 @@ void switch_telnyx_process_audio_stats(switch_core_session_t* session, switch_rt
 	}
 }
 
+void switch_telnyx_on_populate_event(switch_event_t* event)
+{
+	switch_thread_rwlock_rdlock(_rwlock);
+	for (auto iter = _populate_event_callbacks.begin(); iter != _populate_event_callbacks.end(); iter++)
```

Comment:
Use `auto`?

Reply:
Ah ok that was from fixing the merge conflict. I think I can push that change here and will not cause any issues during merge